README.md

```diff
@@ -22,11 +22,21 @@ operator-sdk build quay.io/che-incubator/che-workspace-crd-controller:7.1.0
 
 ### Run controller locally
 1. `kubectl apply -f ./deploy/crds`
-2. Make sure right ingress domain is set in `./deploy/registry/local/ingress.yaml` and execute `kubectl apply -f ./deploy/registry/local`
-3. `operator-sdk up local --namespace <your namespace>`
+2. `kubectl create namespace devworkspace-controller`
+3. Make sure that the right domain is set in `./deploy/controller_config.yaml` and `./deploy/registry/local/ingress.yaml`
+4. `kubectl apply -f ./deploy/registry/local`
+4. `kubectl apply -f ./deploy/controller_config.yaml`
+5. `operator-sdk up local --namespace <your namespace>`
 
 ### Test run controller
 
 1. Take a look samples workspace configuration in `./samples` folder.
 2. Apply any of them by executing `kubectl apply -f ./samples/workspace_java_mysql.yaml -n <namespace>`
-3. As soon as workspace is started you're able to get IDE url by executing `kubectl get workspace -n <namespace>`
\ No newline at end of file
+3. As soon as workspace is started you're able to get IDE url by executing `kubectl get workspace -n <namespace>`
+
+### Run controller locally and debug
+This depends on `delve` being installed (`go get -u github.com/go-delve/delve/cmd/dlv`)
+
+1. Deploy CRDs in same way as "Run controller locally" above
```

Comment:
Is only CRDs enough, or 1-5 steps are needed?

Reply:
Good point, this is unclear.